### DISCUSSION TITLE
Egress Limit

### DISCUSSION THREAD

**Discussion by RejectModders** [supabase/supabase#27063] [Questions]
"Your organization's usage has exceeded its included quota
Your projects can become unresponsive or enter read-only mode. Please upgrade to the Pro plan to ensure that your projects remain available."

"5.242 / 5 GB (105%)"

How we are using so much, is there a way to see whats using this or what..? I am really confused on how we are using so much just for a discord bot..?

It was literally always <%1 but yesterday it started to go up, and its the only thing GOING up. And I have NO clue why.
I am just trying to see what Table could be doing this or what could be causing this stuff, it has never done this before tho.

![image](https://github.com/supabase/supabase/assets/142326052/42524bd3-4231-4a8f-8fb1-cf13e0767c97)

**Comment by dbcaa** [comment]
Get better at supabase duh

**Reply by RejectModders** [reply]
Real

**Comment by ringge** [comment]
I have same issue

**Reply by RejectModders** [reply]
Just self host your on bro, thats what I did, there is no point in using supabase man. You can easily self host your own database. You can have unlimited everything basically. 5gb bandwidth is crazy. That is so low and you can have so much more if you just self host!

**Reply by RejectModders** [reply]
Self host it, I moved from supabase a long time ago. Do NOT use supabase please. It is SO bad.

**Comment by vinayagg** [comment]
I am having same issue. As per advice above, I will likely have to self host it, in which case, I will be wary of using supabase on future projects.

**Reply by RejectModders** [reply]
Self host it, I moved from supabase a long time ago. Do NOT use supabase please. It is SO bad.

**Comment by barhouum7** [comment]
I am also experiencing the same issue, and my project has become unresponsive. The usage limit is actually very low, and at the same time, it can be quickly exceeded. This is merely for a small project!

**Reply by vinayagg** [reply]
Supabase told me it was my problem, couldn't get them to resolve. I moved to postgres.
You can try contacting them to resolve. If you find a resolution, share with everyone here for greater good.
